### DISCUSSION TITLE
Telemetry Issues with Tandem X20HD and X8R receiver

### DISCUSSION THREAD

**Discussion by capajs** [FrSkyRC/ETHOS-Feedback-Community#4573] [Q&A]
Not a total newbie but neither an expert. Been working on FrSky since May, currently preparing a new plane for flight. I have encountered an issue when trying to read Telemetry data from GPS and Vario on the Tandem X20, connected to an X8R receiver. The X8R receiver is set up with 4 servos for 3 controls surfaces and ESC, on S.Port, I have daisy chained a Vario, a GPS, an ASS100 sensor, a FAS150 current sensor and a FLVS ADV voltage sensor, which is connected to the receiver. I last updated my X20HD in August and didn't have any issues back then. I'm not sure whether measuring an 8S battery may be causing the problem since previously I was using a 6S battery. The issue is once I turn on the receiver, it will very briefly read the altitude value from Vario and after 10-15sec it will state "sensor lost" on the Tandem transmitter and GPS will also disconnect but all other sensors continue working. I tried isolating the problem by changing both Vario and GPS of the same type to test them and it happened once again. I have tried looking over in FrSky's website and some other forums but some people state the problem is related to the FLVS sensor which causes the GPS to stop working, but not the Vario. Any help on how to solve said problem is greatly appreciated!

**Comment by Khalid-Hersafril-Flowcopter** [comment]
@capajs I am currently facing the same issue, have you managed to fix it?
